### PR TITLE
Feature/tvist1 126 case status

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: src/Controller/Admin/BoardCrudController.php [hunk 1]
```diff
@@ -55,5 +56,7 @@ public function configureFields(string $pageName): iterable
             ->hideOnForm()
         ;
         yield IntegerField::new('defaultDeadline', 'Default Deadline(days)');
+
+        yield TextareaField::new('statuses', 'Statuses');
```
Comment (jekuaitk) [inline src/Controller/Admin/BoardCrudController.php:60]: Should probably be translated via 'templates/translations/admin.html.twig'

File: src/Controller/CaseProgressController.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        $this->workflowService = $workflowService;
+    }
+
+    public function renderCaseProgressBar(CaseEntity $case): Response
+    {
+        $workflow = $this->workflowService->getWorkflowForCase($case);
+        $places = $workflow->getDefinition()->getPlaces();
+        $currentIndex = array_search($case->getCurrentPlace(), array_keys($places));
+        $progress = floor($currentIndex / sizeof($places) * 100);
```
Comment (jekuaitk) [inline src/Controller/CaseProgressController.php:24]: We briefly discussed this but I'm not sure if we made the correct decision. Currently, last status/place does not result in a full progress bar and the initial status/place shows no progress.

File: src/Form/CaseStatusForm.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            'available_statuses' => [],
+        ]);
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->add('status', ChoiceType::class, [
+            'choices' => $options['available_statuses'],
+            'label' => $this->translator->trans('Change case status', [], 'case'),
+            'placeholder' => $this->translator->trans('Choose new case status', [], 'case'),
```
Comment (jekuaitk) [inline src/Form/CaseStatusForm.php:35]: Missing translation

File: src/Form/CaseStatusForm.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        ]);
+        $builder->add('Gem', SubmitType::class);
```
Comment (jekuaitk) [inline src/Form/CaseStatusForm.php:37]: Should probably be renamed and translated

**Review by jekuaitk — CHANGES_REQUESTED** [review]

File: src/Controller/Admin/BoardCrudController.php (hunk 1 above)
Comment (lats1) [reply src/Controller/Admin/BoardCrudController.php:60]: Fixed in 7398264

File: src/Form/CaseStatusForm.php (hunk 3 above)
Comment (lats1) [reply src/Form/CaseStatusForm.php:35]: Fixed in 7398264

File: src/Form/CaseStatusForm.php (hunk 4 above)
Comment (lats1) [reply src/Form/CaseStatusForm.php]: 7398264

**Review by jekuaitk — APPROVED** [review]

**Pull request merged**